src/Main.java

```diff
@@ -1,5 +1,73 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        // Task 1
+        for (int i = 1; i <= 10; i++)
+            System.out.print(i + " ");
+        divideTasks();
+
+        // Task 2
+        for (int i = 10; i >= 1; i--)
+            System.out.print(i + " ");
+        divideTasks();
+
+        // Task 3
+        for (int i = 0; i <= 17; i += 2)
+            System.out.print(i + " ");
+        divideTasks();
+
+        // Task 4
+        for (int i = 10; i >= -10; i--)
+            System.out.print(i + " ");
+        divideTasks();
+
+        // Task 5
+        for (int year = 1904; year <= 2092; year = year + 8)
```

Comment:
почему шаг 8? должен быть 4
```suggestion
        for (int year = 1904; year <= 2092; year = year + 4)
```

Reply:
фигурные скобки после if/else/for/while надо всегда писать